### ISSUE TITLE
XGBRegressor model stuck in queued status

### ISSUE THREAD

**Issue by jef5ez** [reiinakano/xcessiv#17]
I tried to make a regression model to run on zillow data from kaggle available here https://www.kaggle.com/c/zillow-prize-1/data
Here is a gist of my dataset extractor as well as setting up the XGBRegressor and an exception that was the last thing left in the console
https://gist.github.com/jef5ez/a9b0650293f343682a58b0f0500f3332
I selected the shuffle split for both cross validation settings and added MSE as the learner metric.
The base learner seems to verify fine on the boston housing data.
After hitting finalize and selecting  a single base learner a row shows up below but is stuck in the Queued status.

python 3.5.2
xcessiv (0.2.2)
xgboost (0.6a2)

**Comment by reiinakano** [comment]
Hi! I'll fix this after work. This is a mistake on my part, since I was testing solely on Python 2 and did not account for the fact that `map()` returned an iterator. For now, I suspect things would be fine if you install Xcessiv on Python 2.

**Comment by reiinakano** [comment]
Fixed in release v0.2.5

@jef5ez closing this issue for now but please do open another one if something is still broken. Kaggle competitions are definitely one of the major use cases for Xcessiv
